backend/core/tests/test_model_admin_actions_exist.py

```diff
@@ -0,0 +1,12 @@
+from django.test import TestCase
+
+from core.admin import TeamAdmin
+
+
+class TestModelAdminActionsExist(TestCase):
+     def setUp(self) -> None:
```

Comment:
If you don't make a setUp method then the super class's setup will get called by default. Did you mean to do something here?  This code doesn't do anything.